packages/react/src/primitives/types/textAreaField.ts

```diff
@@ -1,8 +1,10 @@
-import { TextAreaProps } from './textArea';
+import { TextAreaProps, TextAreaStyleProps } from './textArea';
 import { FieldProps } from './field';
 import { FlexContainerStyleProps } from './flex';
 
 export interface TextAreaFieldProps
   extends FieldProps,
     FlexContainerStyleProps,
-    TextAreaProps {}
+    TextAreaProps {
+  inputStyles?: TextAreaStyleProps;
```

Comment:
Not sure if you can do this with interfaces:
```suggestion
  inputStyles?: BaseStyleProps & TextAreaStyleProps;
```